app/assets/stylesheets/application.scss

```diff
@@ -340,7 +348,7 @@ table {
 .spinner-placeholder {
   width: 16px;
   height: 16px;
-  background: image-url('spinner.gif');
+  background: url('spinner.gif');
```

Comment:
it is used in two-pane and also in #2971. we would have to rebase one or the other depends on which one gets merged first.